mobileApp/jobs/buildWhiteLabelAndroidApps.groovy

```diff
@@ -213,8 +214,13 @@ secretMap.each { jobConfigs ->
                     reportFiles('lint-results-prodDebug.html')
                     reportName('Linting Results')
                 }
+                report("\$APP_BASE_DIR/edx-app-android/OpenEdXMobile/build/reports/androidTests/connected/flavors/PROD/") {
+                    allowMissing(false)
+                    keepAll(true)
+                    reportFiles('index.html')
```

Comment:
Are there any supporting files we need to include? (Some things, like coverage, include the index.html with a folder and/or sub-pages and images to include.